code/ColladaParser.cpp

```diff
@@ -297,7 +297,7 @@ void ColladaParser::ReadAnimationClipLibrary()
 				else if (indexID >= 0)
 					animName = mReader->getAttributeValue(indexID);
 				else
-					animName = "animation_" + mAnimationClipLibrary.size();
+					animName = std::string("animation_") + std::to_string(mAnimationClipLibrary.size());
 
```

Comment:
std::to_string( ... ) is a C++11 call ( see http://de.cppreference.com/w/cpp/string/basic_string/to_string ).